app/src/main/java/ch/epfl/sdp/peakar/points/POICacheContent.java

```diff
@@ -0,0 +1,40 @@
+package ch.epfl.sdp.peakar.points;
+
+import org.osmdroid.util.BoundingBox;
+
+import java.util.ArrayList;
+
+/**
+ * Class used for serialization and deserialization
```

Comment:
Maybe it's just me, but this comment made me think that this class contains the method we can use for JSON serialization and deserialization, whereas it just represents the content that is serialized and deserialized to JSON using other methods.
What do you think?

Reply:
Right, i'll correct that